### PR TITLE
Header Template for clean react app

### PR DESCRIPTION
**Pull request by Ashok-Kumar-dharanikota**
- [x] Commit messages are ready to go in the changelog (see below for details)
- [x] PR template filled in (see below for details)

### PR Template

_Please describe what this PR is for, or link the issue that this PR fixes_

issue #17 

The purpose of this issue is to add common sections as a template to this project. So user can use the template directly instead of re-creating common stuff again.

In this PR,

I added two components and wrapped the navigation component inside the Header component and the Header component inside the Template. Users can also access the Header and Navigation components individually. If he wants to create a custom Header and navigation.

folder structure 
```
  presentation/
    components/
      Template/
        Header/
              index.tsx // contains header
              styles.scss
        Navigation/
              index.tsx // contains navigation
              styles.scss
        index.tsx // mount header and navigation and provides a template
        styles.scss
    pages/
      Dashboard/
```

_Thanks again!_

### CODE REVIEW

File: src/presentation/pages/Dashboard/index.tsx [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 type DashboardProps = {}
 
 const Dashboard: React.FC<DashboardProps> = () => {
   return (
-    <div className={Styles.container}>
-      <img src="https://upload.wikimedia.org/wikipedia/commons/a/a7/React-icon.svg" />
-      <span>Start your development from here</span>
-    </div>
+    <Template>
+      <div className={Styles.App}>
```
Comment (rubemfsv) [inline src/presentation/pages/Dashboard/index.tsx]: use the app in lowercase

File: src/presentation/pages/Dashboard/styles.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 @import '../../styles/colors.scss';
 
-.container {
+.App {
```
Comment (rubemfsv) [inline src/presentation/pages/Dashboard/styles.scss]: use the app in lowercase

File: src/presentation/components/Template/Header/styles.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
+.Container {
```
Comment (rubemfsv) [inline src/presentation/components/Template/Header/styles.scss]: use the container in lowercase

File: src/presentation/components/Template/Header/index.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Styles from './styles.scss'
+import Navigation from '../Navigation'
+
+const Header: React.FC = () => {
+  return (
+    <div className={Styles.Container}>
```
Comment (rubemfsv) [inline src/presentation/components/Template/Header/index.tsx]: use the container in lowercase

File: src/presentation/components/Template/Header/index.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+      <img
+        className={Styles.logo}
+        src="https://upload.wikimedia.org/wikipedia/commons/a/a7/React-icon.svg"
```
Comment (rubemfsv) [inline src/presentation/components/Template/Header/index.tsx]: I suggest you to put something written like "My App", as I did here https://main--cheery-shortbread-5efb5b.netlify.app/

it's easier to change

**Review by rubemfsv — CHANGES_REQUESTED** [review]
Could you provide some print screens?

**Comment by Ashok-Kumar-dharanikota** [comment]
Hi @rubemfsv ,

I fixed the changes and this is the result

Dashboard

**Comment by rubemfsv** [comment]
@Ashok-Kumar-dharanikota could you take a look at the failing tests?

**Comment by Ashok-Kumar-dharanikota** [comment]
Hi @rubemfsv ,

I found the reason why it's failing. Due to a naming issue, It is unable to find the file. So I renamed the file name to styles.scss before that I synced the project.

**Review by rubemfsv — APPROVED** [review]

**Pull request merged**